src/bot/commands/random/cat-facts.ts

```diff
@@ -0,0 +1,55 @@
+import { Command, Input, Listener } from '@api';
+import * as request from 'request';
+import { Message } from 'discord.js';
+import { Emoji } from '@bot/libraries/emoji';
+const entities = require("html-entities").AllHtmlEntities;
+
+export class CatFacts extends Command {
+    constructor() {
+        super({
+            name: 'catfact',
+            description: 'Displays a random cat fact.',
+            aliases: ["cfact", "catfacts", "catf"]
+        });
+    }
+
+    async execute(input: Input) {
+        
+        let url = 'https://catfact.ninja/fact?max_length=2000';
+
+        console.log(url);
+        let message = await input.channel.send(`${Emoji.LOADING}  Fetching image...`) as Message;
```

Comment:
```suggestion
        let message = await input.channel.send(`${Emoji.LOADING}  Fetching fact...`) as Message;
```

I guess I will be going through your commands and hardcoding them in the future. A user would not expect a "loading" message for a one-sentence response.